OpenOversight/app/templates/officer.html

```diff
@@ -55,8 +55,8 @@ <h3>General Information
                 <td>{{ officer.gender }}</td>
               </tr>
               <tr>
-                <td><b>Birth Year</b></td>
-                <td>{{ officer.birth_year }}</td>
+                <td><b>Birth Year (Age)</b></td>
+                <td>{{ officer.birth_year }} ({{ officer.birth_year|get_age }})</td>
```

Comment:
We should probably not display `None` in these fields when birth year is not populated, however, that issue was not introduced in this diff, so merging